__tests__/server.test.js

```diff
@@ -122,4 +122,55 @@ describe('GET /api/articles',()=>{
             expect(body.articles).toBeSortedBy('created_at',{descending:true})
         })
     })
+})
+
+describe('GET /api/articles/:article_id/comments',()=>{
+    test('should return a 200 status code with an array of comments',()=>{
+        return request(app)
+        .get('/api/articles/1/comments')
+        .expect(200)
+        .then(({body}) => {
+            expect(body.comments).toHaveLength(11)
+            body.comments.forEach(comment => {
+                expect(typeof comment.comment_id).toBe('number')
```

Comment:
Nice! This is a perfectly valid way to handle this. If you wanted to refactor, you could also look at what Jest matchers are available to test for objects 👀